extensions/ql-vscode/test/unit-tests/databases/config/db-config-store.test.ts

```diff
@@ -184,6 +185,146 @@ describe("db config store", () => {
     });
   });
 
+  describe("add db items", () => {
+    let app: App;
+    let configPath: string;
+
+    beforeEach(async () => {
+      app = createMockApp({
+        extensionPath,
+        workspaceStoragePath: tempWorkspaceStoragePath,
+      });
+
+      configPath = join(tempWorkspaceStoragePath, "workspace-databases.json");
+    });
+
+    it("should add a remote repository", async () => {
+      // Initial set up
+      const dbConfig = createDbConfig();
+
+      await writeJSON(configPath, dbConfig);
```

Comment:
(Optional) We could extract this and the next 2 lines into a function and re-use across all the tests.

Reply:
Good idea! Done!